src/gov/nasa/worldwind/layers/AbstractGraticuleLayer.java

```diff
@@ -721,9 +721,8 @@ protected Object createLineRenderable(Iterable<? extends Position> positions, St
     {
         Path path = new Path(positions);
         path.setPathType(pathType);
-        path.setFollowTerrain(true);
+        path.setSurfacePath(true);
```

Comment:
What's the reasoning behind this change? Isn't the `setAltitudeMode` usage more illustrative of the different modes that a `Path` may use?

Reply:
I see that this pattern is changed in other classes as well, for instance the stuff in the `milstd2525`.

Reply:
Polyline (deprecated) and Path (which replaces it) have different treatments of followTerrain. The Path class was designed several years ago and I didn't want to modify its behavior.

I initially replaced polyline.setFollowTerrain with path.setFollowTerrain, but noticed that the behavior was different. Polyline implicitly also applies clamp_to_ground behaviour with followTerrain while path does not. 

So rather than replace setFollowTerrain with two calls to achieve the same behavior, I expanded on the isSurfacePath pattern in the Path class to set followTerrain and clamp_to_ground in one call.